habitat-hitl/habitat_hitl/core/client_helper.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+from habitat_hitl.core.average_helper import AverageHelper
+
+
+class ClientHelper:
+    """
+    Tracks connected remote clients. Displays client latency and kicks idle clients.
+    """
```

Comment:
I like that we're grouping this functionality here.

Nit: the name `ClientHelper` is vague in terms of what this class actually does.

Reply:
Yeah but I figure the functionality will grow.